### PR TITLE
Refactor entry exports with match statements

### PR DESCRIPTION
**Pull request by syucream**
SSIA

### CODE REVIEW

File: entity/api_v2/views.py [hunk 1]
```diff
@@ -297,7 +297,7 @@ def get_queryset(self):
             return entity_attrs.values_list("name", flat=True).order_by("name").distinct()
         # single parent_entity case
         else:
-            return entity_attrs.values_list("name", flat=True).order_by("index")
+            return entity_attrs.values_list("name", flat=True).order_by("index", "pk")
```
Comment (syucream) [inline entity/api_v2/views.py:300]: Its a not core change but it fixes a flaky test case. The latest test result in the master branch has the same cause https://github.com/dmm-com/airone/actions/runs/8641820682/job/23691959470

**Review by hinashi — APPROVED** [review]
LGTM!

File: entity/api_v2/views.py (hunk 1 above)
Comment (userlocalhost) [reply entity/api_v2/views.py:300]: Thank you very much for fixing it 👍

**Review by userlocalhost — APPROVED** [review]
Thank you very much for your great work every time 👍

**Pull request merged**